telescope/selector_test.py

```diff
@@ -198,6 +324,7 @@ def testEncodeMultiSelectorOneElement(self):
     s.duration = 45
     s.site_names = ['mia01']
     s.client_providers = ['twc']
+    s.client_countries = ['us', 'ca', 'uk']
```

Comment:
This is my fault for terrible naming, but the idea of this test is that for every field in the multiselector whose type is a list, the list only contains one element.

Can you change this to `s.client_countries = ['us']` and move this line to `testEncodeMultiSelectorMultiElement` and if you can think of a better test name that makes the intent clearer, please change it.